### PR TITLE
ovirt vm update kernel docs

### LINKED ISSUE #60383: ovirt: linux boot params removal
##### SUMMARY

linux boot parameters for a VM is going to be removed due to:
https://bugzilla.redhat.com/1732437
The ansible should not be able to invoke these options.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

ovirt_vm
ovirt_instance_type

##### ANSIBLE VERSION

```paste below
ansible 2.8.3
```

##### CONFIGURATION

```paste below
Default
```

##### STEPS TO REPRODUCE

```yaml
---

- name: Update a VM
  hosts: all
  tasks:
    - block:
        - name: Login to oVirt
          ovirt_auth:
            url: "{{ api_url }}"
            username: "{{ username }}"
            password: "{{ password }}"
            insecure: "{{ insecure }}"
          when: ovirt_auth is undefined or not ovirt_auth
          register: loggedin

        - name: Change linux boot params
          ovirt_vm:
            auth: "{{ ovirt_auth }}"
            name: test
            operating_system: other_linux
            kernel_path: iso://initramfs.img
      always:
        - name: Logout from oVirt
          ovirt_auth:
            state: absent
            ovirt_auth: "{{ ovirt_auth }}"
          when: not loggedin.skipped | default(false)

```

##### EXPECTED RESULTS

Should fail with unsupported type.

##### ACTUAL RESULTS

At the moment the operation functioning. In ovirt 4.4 soon it won't be supported.

```paste below

```

### PR DESCRIPTION
**Pull request by mnecas**
##### SUMMARY

ovirt vm update kernel docs

Fixes #60383
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

ovirt
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 1]
```diff
@@ -522,21 +522,25 @@
         description:
             - "If I(true) C(kernel_params), C(initrd_path) and C(kernel_path) will persist in virtual machine configuration,
                if I(False) it will be used for run once."
+            - Not compatible with ovirt engine version 4.4+.
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: I'd rephrase: `Parameter is usable with oVirt 4.3 and lower, but it's going to be removed in oVirt 4.4`

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 2]
```diff
@@ -522,21 +522,25 @@
...
         type: bool
         version_added: "2.8"
     kernel_path:
         description:
             - Path to a kernel image used to boot the virtual machine.
             - Kernel image must be stored on either the ISO domain or on the host's storage.
+            - Not compatible with ovirt engine version 4.4+.
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: Same as above

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 3]
```diff
@@ -522,21 +522,25 @@
...
         version_added: "2.3"
     initrd_path:
         description:
             - Path to an initial ramdisk to be used with the kernel specified by C(kernel_path) option.
             - Ramdisk image must be stored on either the ISO domain or on the host's storage.
+            - Not compatible with ovirt engine version 4.4+.
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: Same as above

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 4]
```diff
@@ -522,21 +522,25 @@
...
         version_added: "2.3"
     kernel_params:
         description:
             - Kernel command line parameters (formatted as string) to be used with the kernel specified by C(kernel_path) option.
+            - Not compatible with ovirt engine version 4.4+.
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: Same as above

**Review by mwperina — CHANGES_REQUESTED** [review]
Ondro, do you think it's worth to mark those parameters as deprecated?

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py [hunk 5]
```diff
@@ -522,21 +522,25 @@
...
             - "If I(true) C(kernel_params), C(initrd_path) and C(kernel_path) will persist in virtual machine configuration,
                if I(False) it will be used for run once."
+            - Parameter is usable with oVirt 4.3 and lower, but it's going to be removed in oVirt 4.4
         type: bool
         version_added: "2.8"
     kernel_path:
         description:
             - Path to a kernel image used to boot the virtual machine.
             - Kernel image must be stored on either the ISO domain or on the host's storage.
+            - Parameter is usable with oVirt 4.3 and lower, but it's going to be removed in oVirt 4.4
```
Comment (acozine) [inline lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: We are trying to eliminate contractions (we're, it's, isn't) to make translating the docs easier. Also, description lines should end with a period. Please change all three lines to:

```suggestion
            - Usable with oVirt 4.3 and lower; removed in oVirt 4.4.
```

**Review by acozine — COMMENTED** [review]
Thanks @mnecas for keeping up with changes to oVirt! Could you update the wording as suggested below? Thanks.

File: lib/ansible/modules/cloud/ovirt/ovirt_vm.py (hunk 5 above)
Comment (mnecas) [reply lib/ansible/modules/cloud/ovirt/ovirt_vm.py]: Thank you for change recommendations.

**Review by mwperina — APPROVED** [review]
shipit
